resources/sdk/pureclouddotnet/extensions/Model/LocalDate.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using {{= it.packageName}}.Model;
+
+namespace {{=it.packageName }}.Model
+{
+  /// <summary>
+  /// The LocalDate class is a wrapper class that maps the Genesys Cloud swagger definition to a Date only class.
+  /// This class written to deal with DEVENGAGE-338 where a customer raised an issue where they were getting improperly
+  /// formatted Dates (e.g. with a timestamp) because the default serialization for a LocalDate was to serialize to a 
+  /// DateTime.
+  /// </summary>  
+public class LocalDate
+{
+  private DateTime dateTime;
+
+  public LocalDate(DateTime dateTime)
```

Comment:
I believe this needs a default constructor and the dateTime property needs to be public so this class can be used for deserializing responses from the API as well.